client/dashboard/plugins/scheduled-updates/new/components/plugins-selection/index.tsx

```diff
@@ -0,0 +1,77 @@
+import { DataViews, Field, View, filterSortAndPaginate, type Action } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { useMemo, useState } from 'react';
+import { DataViewsCard } from '../../../../../components/dataviews-card';
+import { useEligiblePlugins } from '../../../hooks/use-eligible-plugins';
+import type { CorePlugin } from '@automattic/api-core';
+
+const pluginFields: Field< CorePlugin >[] = [
+	{
+		id: 'name',
+		label: __( 'Plugin' ),
+		enableGlobalSearch: true,
+		render: ( { item } ) => item.name,
+		getValue: ( { item } ) => item.name,
+	},
+];
+
+type Props = {
+	selectedSiteIds: string[];
+	selection: string[];
+	onChangeSelection: ( slugs: string[] ) => void;
+};
+
+function ScheduledUpdatesPluginsSelection( {
+	selectedSiteIds,
+	selection,
+	onChangeSelection,
+}: Props ) {
+	const eligiblePlugins = useEligiblePlugins( selectedSiteIds );
+	const [ view, setView ] = useState< View >( {
+		type: 'table',
+		page: 1,
+		perPage: 5,
+		sort: { field: 'name', direction: 'asc' },
+		fields: [],
+		titleField: 'name',
+	} );
+	const { data: filtered, paginationInfo } = useMemo( () => {
+		return filterSortAndPaginate< CorePlugin >(
+			eligiblePlugins as CorePlugin[],
```

Comment:
Isn't `eligiblePlugins` of type Array<CorePlugin> already? Can we remove the casting?